development-vm/fix-elasticsearch-2.4.sh

```diff
@@ -19,7 +19,7 @@ if [ -d "/usr/share/elasticsearch/plugins/migration" ]; then
 fi
 
 echo "Removing Elasticsearch data directory"
-echo "This is required as the data needs to be rebuilt for ES 2.4"
+echo "This is require as the data needs to be rebuilt for ES 2.4"
```

Comment:
If you're fixing typos, there's an "elastiocsearch" on line 4 that it'd be good to catch. 🙂

Reply:
didn't do this as I forgot I wanted to merge this PR before I sent the note out and waiting for this to rebuild might have caused problems. I will try and remembner to do this next time I have a puppet PR (later today)